server/rootless.go

```diff
@@ -0,0 +1,37 @@
+package server
+
+import (
+	"strings"
+
+	rspec "github.com/opencontainers/runtime-spec/specs-go"
+	"github.com/opencontainers/runtime-tools/generate"
+)
+
+func makeOCIConfigurationRootless(g *generate.Generator) error {
+	g.Config.Linux.Resources = nil
+	g.Config.Process.OOMScoreAdj = nil
```

Comment:
I'm not familiar with apparmor yet but probably it does not work with rootless?

Reply:
I am not familiar either but I guess it won't work, similarly with SELinux, I'll force them to ""

Reply:
SELinux would work in UserNS.  SElinux doesn't care about DAC.